privacyidea/lib/policy.py

```diff
@@ -2021,77 +2024,100 @@ def get_static_policy_definitions(scope=None):
                 'type': 'str',
                 'desc': _("Allow the user to authenticate (default). If set to '{0!s}', "
                           "the authentication of the user will be denied.").format(AUTHORIZED.DENY),
-                'value': [AUTHORIZED.ALLOW, AUTHORIZED.DENY]
+                'value': [AUTHORIZED.ALLOW, AUTHORIZED.DENY],
+                'group': GROUP.MODIFYING_RESPONSE,
             },
             ACTION.APPLICATION_TOKENTYPE: {
                 'type': 'bool',
                 'desc': _("Allow the application to choose which token types should be used "
                           "for authentication. Application may set the parameter 'type' in "
                           "the request. Works with validate/check, validate/samlcheck and "
-                          "validate/triggerchallenge.")
+                          "validate/triggerchallenge."),
+                'group': GROUP.CONDITIONS,
```

Comment:
I am sorry. This is actually a policy that allows the client to choose a tokentype. 
So this is rather not a condition.
I think it is...
...nothing. So I would leave the at misecellaneous. Please remove the group.

Reply:
done in 8ce2e724